views.py

```diff
@@ -0,0 +1,169 @@
+from flask import render_template, session, redirect, request
+from datetime import datetime
+
+from init import db, app
+from forms import LoginForm, RegistrationForm, OrderForm
+from tables import User, Dish, Category, Order
+from words import correct_word
+
+
+@app.route("/")
+def render_main():
+    items = db.session.query(Dish)
+    categories = db.session.query(Category)
+    session["word"] = correct_word(len(session.get("cart", [])))
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    return render_template("main.html", categories=categories, items=items, dishes=session.get("cart", []), user=user)
+
+
+@app.route("/cart/", methods=["GET", "POST"])
+def render_cart():
+    flag = 0
+    form = OrderForm()
+    cart = session.get("cart", [])
+    use_it = set()
+    new_cart = []
+    for item in cart:
+        if item not in use_it:
+            new_cart.append(item)
+            use_it.add(item)
+    user = None
+    if session.get("user_id"):
+        user = User.query.filter(User.id == session.get("user_id")).first()
+    session["word"] = correct_word(len(session.get("cart", [])))
+    return render_template("cart.html", form=form, new_cart=new_cart, flag=flag, user=user)
+
+
+@app.route("/add/<item>/")
+def add_to_cart(item):
+    flag = 0
+    form = OrderForm()
+    items = db.session.query(Dish)
```

Comment:
That's odd. It's really bad practice and really slow thing to do. 
If I get it right you load whole table into memory an search through it in your code...
There are `select ... where` queries for that. rework this pls

Reply:
If I not mistaking, its should be work like "select ... where" inside.

Reply:
I take from all the tables of dishes right dish, that I need.

Reply:
Yeah, no. As I can tell, you just take the whole table and then filter through it manually.
What you'd want to use is something like `s = select(users, addresses).where(users.c.id == addresses.c.user_id)`